pkg/config/config.go

```diff
@@ -163,19 +158,20 @@ func initMock() {
 	h.SetupMockAWS()
 	h.SetupMockGCE()
 
+	ctx := context.Background()
```

Comment:
Why not propagate the context received in `ConfigureProvider` ?

Reply:
Same here! Thanks!